conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/DialogueFeignClient.java

```diff
@@ -88,20 +94,22 @@ final class DialogueFeignClient implements feign.Client {
     @Override
     public feign.Response execute(Request request, Request.Options _options) throws IOException {
         com.palantir.dialogue.Request.Builder builder = com.palantir.dialogue.Request.builder();
-        Optional<RequestBody> body = requestBody(request);
-        if (body.isPresent()) {
-            builder.body(body);
-            builder.putHeaderParams(HttpHeaders.CONTENT_LENGTH, Integer.toString(request.body().length));
-        }
```

Comment:
`RequestBody` got a `contentLength` method in https://github.com/palantir/dialogue/pull/1227, so we can move this responsibility into Dialogue.